qiita_pet/templates/study_description_templates/prep_template_info_tab.html

```diff
@@ -188,5 +188,19 @@ <h4 class="panel-title">
         </td>
       </tr>
       {% end %}
+      {{is_editable}}
+    {% if is_editable %}
+      <tr><td><hr></td></tr>
+      <tr>
+        <td>
+          <select id="update_prep_template_{{pt_id}}">
+            {% for f in files %}
```

Comment:
This is currently showing all files. Probably useful to filter them and show only the ones than end with txt or tsv, as is done in other places. (This needs to be changed in the PrepTemplateInfoTab class)